integration

```diff
@@ -564,6 +564,7 @@
   "macxq/foxess-ha",
   "madpilot/hass-amber-electric",
   "make-all/metlink-nz",
+  "mampfes/ha_bayernluefter",
```

Comment:
This is for Germany only, and needs a country key in hacs.json https://github.com/mampfes/ha_bayernluefter/blob/main/hacs.json

Reply:
fixed